src/integrationTest/java/org/opensearch/security/api/ConfigRestApiIntegrationTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+package org.opensearch.security.api;
+
+import java.util.StringJoiner;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.junit.Test;
+
+import org.opensearch.security.DefaultObjectMapper;
+import org.opensearch.security.dlic.rest.api.Endpoint;
+import org.opensearch.test.framework.cluster.TestRestClient;
+
+import static org.opensearch.security.api.PatchPayloadHelper.patch;
+import static org.opensearch.security.api.PatchPayloadHelper.replaceOp;
+import static org.opensearch.security.dlic.rest.api.RestApiAdminPrivilegesEvaluator.SECURITY_CONFIG_UPDATE;
+import static org.opensearch.security.support.ConfigConstants.SECURITY_RESTAPI_ADMIN_ENABLED;
+import static org.opensearch.security.support.ConfigConstants.SECURITY_UNSUPPORTED_RESTAPI_ALLOW_SECURITYCONFIG_MODIFICATION;
+
+public class ConfigRestApiIntegrationTest extends AbstractApiIntegrationTest {
+
+    final static String REST_API_ADMIN_CONFIG_UPDATE = "rest-api-admin-config-update";
+
+    static {
+        clusterSettings.put(SECURITY_UNSUPPORTED_RESTAPI_ALLOW_SECURITYCONFIG_MODIFICATION, true).put(SECURITY_RESTAPI_ADMIN_ENABLED, true);
+        testSecurityConfig.withRestAdminUser(REST_ADMIN_USER, allRestAdminPermissions())
+            .withRestAdminUser(REST_API_ADMIN_CONFIG_UPDATE, restAdminPermission(Endpoint.CONFIG, SECURITY_CONFIG_UPDATE));
+    }
+
+    private String securityConfigPath(final String... path) {
+        final var fullPath = new StringJoiner("/").add(super.apiPath("securityconfig"));
```

Comment:
will these tests be run against both: `/_opendistro/` and `/_plugins/` or will they be randomly for one of these during any given run?

Reply:
yes it is. it calls `super.apiPath`

Reply:
from what i understand it will randomly pick 1 during any given run and since we run tests 1 time, the endpoint picked could be either opendistro or plugins.

Reply:
Exactly per each request

Reply:
Besides each REST API integration test uses 1 or 3 nodes clusters.

Reply:
okie.